cmd/user-agent/src/main.js

```diff
@@ -84,7 +84,9 @@ let defaultTrustBlocStartupOpts = {
     assetsPath: '/trustbloc-agent/assets',
     blocDomain: 'testnet.trustbloc.local',
     walletMediatorURL: '',
-    'log-level': 'debug'
+    'log-level': 'debug',
```

Comment:
We should stop adding properties that need quotes in javascript.

Reply:
the `log-level` one was from a previous commit, so I left it alone for the sake of keeping this PR more focused on the SDS stuff, but I just updated the agentUsername and SDSServerURL ones I just added to not have quotes.